### ISSUE TITLE
Template problem

### ISSUE THREAD

**Issue by chrizzbee** [jelovirt/com.elovirta.ooxml#53]
Hi Jarno
I want modify the Normal.dotx that comes with the plugin. When I open and save it under a new name and use this new template in built.xml, the output doesn't look like as with the original Normal.dotx. For instance, all heading styles are gone and so are the table borders. The size of the template decreases from 78 KB to 28 KB.

Which Word version is recommended to make template modifications? I use Word 2010. I tried Libre Writer, but then the result can not be openend by Word.

Thanks
Chris.

**Comment by chrizzbee** [comment]
Ok, I figured out, that a German Word doesn't go well with en English template. Because all styles are translated to German upon save, this scrambled the output. I downloaded an English language pack and now it works like a charm ...

**Comment by jelovirt** [comment]
Yes, differing template style names will cause issues. The mapping which style is used uses the style name/label, because the internal ID for the style can change. Using an alternative mapping would require an XSLT override.

**Comment by jelovirt** [comment]
The documentation is at https://github.com/jelovirt/com.elovirta.ooxml/wiki/Templates but it’s not particularly good. Enhancement contributions are welcome.

**Comment by rag-sing** [comment]
Hi all,

I used latest DITA OT 3.6.1 and com.elovirta.ooxml plugin. I have been successful getting output. But I got one problem "The next chapter starts right below where the previous chapter ends" in the docx output.

But as per my understanding the next chapter should starts next page where the previous chapter ends.

Please suggest which xsl file. I need to update.

Thanks,
Raghavendra
